modules/process/SV/SomaticMergeSVs.nf

```diff
@@ -1,15 +1,13 @@
 process SomaticMergeSVs {
   tag "${idTumor}__${idNormal}"
 
-  publishDir "${params.outDir}/somatic/${outputPrefix}/combined_svs", mode: params.publishDirMode, pattern: "*.merged.vcf.{gz,gz.tbi}"
-  publishDir "${params.outDir}/somatic/${outputPrefix}/combined_svs/intermediate_files", mode: params.publishDirMode, pattern: "*.merged.clean.vcf"
+  publishDir "${params.outDir}/somatic/${outputPrefix}/combined_svs/intermediate_files", mode: params.publishDirMode, pattern: "*.merged.vcf.{gz,gz.tbi}"
+  publishDir "${params.outDir}/somatic/${outputPrefix}/combined_svs/intermediate_files", mode: params.publishDirMode, pattern: "*.merged.raw.vcf"
```

Comment:
Suggest to make `*.merged.raw.vcf.gz` and `*.merged.raw.vcf.gz.tbi` file just in case they are too big unzipped

Reply:
done, no issues with this